TAF_EasyRest/TestFramework/PageComponents/Owner/CreateMenuPageComponents/CreateListMenuAndComponent.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+namespace TestFramework.PageComponents.Owner.CreateMenuPageComponents
+{
+    public class CreateListMenuAndComponent
+    {
+            IWebDriver driver { get; }
+
+            public CreateListMenuAndComponent(IWebDriver driver)
+            {
+                this.driver = driver;
+            }
+
+            private IWebElement _finishButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[2]"));
+            private IWebElement _backButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[1]"));
+
+            public CreateListMenuAndComponent ClickThreeDotsButton()
```

Comment:
The name is ThreeDotsButton, but you click on finish button inside the method. Please make changes.